client/src/components/HistoryViewer/HistoryViewerToolbar.js

```diff
@@ -7,18 +7,16 @@ import { connect } from 'react-redux';
 import i18n from 'i18n';
 import { inject } from 'lib/Injector';
 import { addMessage, showList } from 'state/historyviewer/HistoryViewerActions';
-import { Tooltip } from 'reactstrap';
+import { UncontrolledTooltip as Tooltip } from 'reactstrap';
```

Comment:
The state control we were doing here was pointless - it was just toggling a boolean in state. No need for us to manage that directly.